src/content/docs/browser/new-relic-browser/browser-apis/setcustomattribute.mdx

```diff
@@ -72,17 +80,33 @@ Errors for custom attributes will be included in events on the [JS Errors page](
 
     <tr>
       <td>
-        `$value`
+        `value`
 
-        _string_ OR _integer_
+        _string_ OR _integer_ OR _null_
       </td>
 
       <td>
         Required. Value of the attribute. Appears as the value in the named attribute column in the `PageView` event. It will appear as a column in the `PageAction` event if you are using it. Custom attribute values cannot be complex objects, only simple types such as Strings and Integers.
 
+        Passing a `null` value will act as an un-setting of any existing attribute by the same name.
+
         Avoid using [reserved NRQL words](/docs/insights/new-relic-insights/adding-querying-data/inserting-custom-events-attributes-insights-javascript-api#limits) when you name the attribute/value.
       </td>
     </tr>
+
+    <tr>
+      <td>
+        `persist`
+
+        _boolean_
+      </td>
+
+      <td>
+        Optional. If set to `true`, the name-value pair will also be set into the browser's storage API, and on the following instrumented page loads that is within the same session, the pair will be re-applied as a custom attribute.
```

Comment:
@cwli24 I didn't quite understand this sentence. Could we divide this into two like this: "If set to `true`, the name-value pair will also be set in the browser's storage API. On the following instrumented page that loads within the same session, the pair will be re-applied as a custom attribute.

I'm not quite sure I have a good revision for the second part. Please double check this.